app/script/view_model/content/InputBarViewModel.js

```diff
@@ -179,6 +174,13 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
       this.conversationHasFocus(true);
       this.pastedFile(null);
       this.cancelMessageEditing();
+      if (this.conversationEntity()) {
+        const mentions = this.conversationEntity().input().mentions;
+        this.currentMentions = mentions;
+        if (this.currentMentions.length) {
+          this.input(`${this.input()} `);
```

Comment:
We need to add a space here or the current mentions index will be off by 1.

Reply:
Can you please explain why? This sounds weird and hacky.

Reply:
No, sorry, I can't explain this but probably @AndyLnd can.

Reply:
Ok seems like it's not related to the space. The index is still sometimes off by 1 when loading from local storage.

Reply:
My guess would be that this is probably to trigger an observable change somewhere and the side effects that we want with it. Depending on what this is we might want to explicitly trigger them..?